client/src/css/NavBar.css

```diff
@@ -83,7 +83,6 @@
 @media only screen and (max-width: 414px) {
   .nav {
     height: 12vh;
-    max-width: 414px;
```

Comment:
Any reason you removed this in the NavBar?

Reply:
I remember an issue with the navbar extending far past the viewport width, and that's why that was there. I forgot to remove it in the earlier commit.

Reply:
Makes sense, Feel free to resolve this when you push your next commit removing it!